src/sentry/event_manager.py

```diff
@@ -68,8 +68,28 @@ def get_hashes_for_event(event):
         result = interface.compute_hashes(event.platform)
         if not result:
             continue
-        return map(md5_from_hash, result)
-    return [md5_from_hash([event.message])]
+        return result
+    return [[event.message]]
+
+
+def get_hashes_from_fingerprint(event, fingerprint):
+    default_values = set(['{{ default }}', '{{default}}'])
```

Comment:
this is intentionally simplistic to avoid slowing things down simply for future support